src/Bundle/Doctrine/ORM/Driver.php

```diff
@@ -34,12 +34,15 @@ public function __construct(ManagerRegistry $managerRegistry)
     public function getDataSource(array $configuration, Parameters $parameters): DataSourceInterface
     {
         if (!array_key_exists('class', $configuration)) {
-            throw new \InvalidArgumentException('"class" must be configured.');
+            throw new \InvalidArgumentException('Missing configuration: when using the ORM driver for a grid, you must define the "class" option.');
         }
 
-        /** @var ObjectManager $manager */
         $manager = $this->managerRegistry->getManagerForClass($configuration['class']);
 
+        if (null === $manager) {
+            throw new RuntimeException(sprintf('Doctrine ORM manager for class "%s" not found.', $configuration['class']));
```

Comment:
@stlgaits what do you think?
```suggestion
            throw new RuntimeException(sprintf('Doctrine ORM manager for class "%s" was not found.', $configuration['class']));
```
Dmitri has suggested the previous sentence, but there's no verb.

Reply:
I think it's ok without a verb, it's the same as "404 - Page not found"